olive/exception/__init__.py

```diff
@@ -0,0 +1,26 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+class OliveException(Exception):
+    """
+    Base class for Olive exceptions.
+    """
+
+    pass
+
+
+class OlivePassException(OliveException):
+    """
+    Base class for Olive pass exceptions.
+    """
+
+    pass
+
+
+class OliveEvaluationException(OliveException):
```

Comment:
Will we need the evaluation exception class? As there is no any `prune` mechanism in evaluation part.

Reply:
this is used to raise if there are some failure when evaluation model.